.php_cs.dist

```diff
@@ -0,0 +1,28 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([__DIR__])
+    ->exclude(['vendor', 'var'])
+    ->notPath('/cache/')
+;
+
+return PhpCsFixer\Config::create()
+    ->setFinder($finder)
+    ->setRules([
+        '@PSR2' => true,
+        '@Symfony' => true,
```

Comment:
Where this config comes from? Why Symfony?

Reply:
These two rules are aliases for a group of rules. Just a proposal :)
If you like the base rules in the `@Symfony` rule then ok, otherwise I can remove that one

Reply:
Okay, lets leave it as it is